app/design/adminhtml/default/openmage/template/login.phtml

```diff
@@ -31,48 +31,49 @@
+            <form method="post" action="<?php echo $this->getUrl('adminhtml') ?>" id="loginForm">
                 <div class="login-form">
                     <input name="form_key" type="hidden" value="<?php echo $this->getFormKey() ?>" />
                     <h2><?php echo Mage::helper('adminhtml')->__('Log in to Admin Panel') ?></h2>
                     <div id="messages">
                         <?php echo $this->getMessagesBlock()->toHtml() ?>
                     </div>
-                    <div class="input-box"><label for="username"><?php echo Mage::helper('adminhtml')->__('User Name:') ?></label><br/>
-                        <input type="text" id="username" name="login[username]" value="" class="required-entry input-text" /></div>
-                    <div class="input-box"><label for="login"><?php echo Mage::helper('adminhtml')->__('Password:') ?></label><br />
-                        <input type="password" id="login" name="login[password]" class="required-entry input-text" value="" /></div>
-                    <?php echo $this->getChildHtml('form.additional.info'); ?>
+                    <div class="input-box input-left">
+                        <label for="username"><?php echo Mage::helper('adminhtml')->__('User Name:') ?></label><br />
+                        <input type="text" id="username" name="login[username]" value="" class="required-entry input-text" autofocus="autofocus" />
+                    </div>
+                    <div class="input-box input-right">
+                        <label for="login"><?php echo Mage::helper('adminhtml')->__('Password:') ?></label><br />
+                        <input type="password" id="login" name="login[password]" value="" class="required-entry input-text" />
+                    </div>
+                    <?php echo $this->getChildHtml('form.additional.info') ?>
                     <div class="clear"></div>
                     <div class="form-buttons">
                         <a class="left" href="<?php echo Mage::helper('adminhtml')->getUrl('adminhtml/index/forgotpassword', array('_nosecret' => true)) ?>"><?php echo Mage::helper('adminhtml')->__('Forgot your password?') ?></a>
-                        <input type="submit" class="form-button" value="<?php echo Mage::helper('core')->quoteEscape(Mage::helper('adminhtml')->__('Login')) ?>" title="<?php echo Mage::helper('core')->quoteEscape(Mage::helper('adminhtml')->__('Login')) ?>" /></div>
+                        <input type="submit" class="form-button" value="<?php echo Mage::helper('core')->quoteEscape(Mage::helper('adminhtml')->__('Login')) ?>" title="<?php echo Mage::helper('core')->quoteEscape(Mage::helper('adminhtml')->__('Login')) ?>" />
+                    </div>
                 </div>
-                <p class="legal"><?php echo Mage::helper('adminhtml')->__('OpenMage LTS') ?></p>
+                <p class="legal"><?php echo Mage::helper('adminhtml')->__('Powered by OpenMage') ?></p>
```

Comment:
You should decide if it is "OpenMage" or "OpenMage LTS". Having already two rows based on "OpenMage LTS" I would change this line to "Powered by OpenMage LTS". If you add LTS don't forget to change in translation this record.

Reply:
Oké!

Reply:
From what I checked you opted for OpenMage LTS. It is good to keep all changes the same. I was less interested in whether it was with or without LTS.